datasets/br/br_tcu_disqualified/crawler.py

```diff
@@ -0,0 +1,92 @@
+from typing import List
+from zavod import Context, helpers as h
+
+
+def fetch_data(context: Context) -> List[dict]:
+    """
+    Fetches data from the website, or raises an exception on failure.
+
+    :param context: The context object.
+    :return: List of dicts containing data.
+    """
+    response = context.fetch_json(context.data_url)
+    if "items" not in response:
+        context.log.error("Items not found in JSON")
+        return []
+    return response["items"]
+
+
+def create_entity(input_dict: dict, context: Context):
+    """
+    Creates an entity from the raw data.
+
+    :param input_dict: Data dict extracted from source.
+                       Should have at least following keys:
+                        - cpf
+                        - nome
+    :param context: The context object.
+    """
+
+    entity = context.make("Person")
+    # the CPF comes in formatted form (i.e. with punctuaction, XXX.XXX.XXX-XX)
+    # and it's common practice to remove this punctuation when saving in the database
+    raw_tax_number = input_dict["cpf"]
+    tax_number = raw_tax_number.replace(".", "").replace("-", "")
```

Comment:
should we create a clean_br_cpf function somewhere and use it in the bcb crawlers too? that could go in zavod.helpers.text to start with until @pudo suggests a better place

Reply:
I think it would be a good idea, they sometimes put the punctuation because it's easier to read with it

Reply:
Do you want me to create a PR for that? I'm guessing it's [here](https://github.com/opensanctions/opensanctions/tree/main/zavod/zavod/helpers), but should I create a separate file?

Reply:
yeah - I think in the `text` module in there to start with, and then exposed under `helpers.` via the __init__ module in that directory

Reply:
this PR or new PR are both good, thank you!

Reply:
I added to this PR, let me know if I did it correctly. I also added the one for CNPJ that will be needed for future crawlers.